src/pyggdrasil/tree_inference/_file_id.py

```diff
@@ -208,15 +208,25 @@ def from_str(cls, str_id: str):
         """
         # split string by underscore and assign to attributes
         # CS_1-T_d_10-100_0.01_0.01_0.01_true_UXR
-        cs_par1, tree_id, cs_part2 = str_id.split("-")
+        parts = str_id.split("-")
+        cs_part1 = parts[0]
+        tree_id = parts[1]
+        print(tree_id)
+        cs_part2 = "-".join(parts[2:])
+
```

Comment:
Allowing for scientific notation error rates. 

The trouble with string matching.

Reply:
Yes, I completely see how one could do this now! Understand Snakemake better now. Would do things differently now :)  Would have made the output file names more human-readable. 

Would be quite a bit of refactoring in `workflows`, `tests` and `tree_inference` to change this now.

I don't see any danger of the error rates representation as numbers breaking down now. 

I'll stick to the current solution. This is probably something I won't change after running the first experiments.

If final output files get tiring to read I write a rule to rename files for ease of finding the files one wants.